keyboards/xd87/xd87.h

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2018 Alexander Fougner <fougner89 at gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#ifndef XD87_H
+#define XD87_H
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+// Keymap utilizing all the possible keys on the PCB.
+#define LAYOUT_7bit( \
```

Comment:
If this supports all possible keys on the PCB, this should be `LAYOUT_all`, i think.

Reply:
Also, this board supports ANSI as well, correct? 

If so, could you add a layout for that, that matches the default community layout? 
https://github.com/qmk/qmk_firmware/blob/master/layouts/default/tkl_ansi/default_tkl_ansi/keymap.c